### ISSUE TITLE
Need to add a way to remove data on DB.

### ISSUE THREAD

**Issue by seanbarclay** [cloudant-labs/backbone.cloudant#2]
Can't remove data from server using model.destroy(). You get a 409 error because you need to specify the _rev.

**Comment by larsparendt** [comment]
I've had success deleting models like this:

```
var model = someCollection.get(modelByID);
var options = model.url() + "\?rev\="+model.get("value")["_rev"];
model.destroy(options);
```

However, I have to launch my browser (Chrome) with security disabled because Cloudant (I think) does not allow OPTIONS in HTTP requests.

**Comment by drsm79** [comment]
Fixed in commit cc8443d303975e21a992a55086125942dba6c8da - sorry for the delay!
